scripts/k8s_releases.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+from typing import List, Optional
+
+import requests
+from packaging.version import Version
+
+K8S_TAGS_URL = "https://api.github.com/repos/kubernetes/kubernetes/tags"
+
+
+def _url_get(url: str) -> str:
+    r = requests.get(url, timeout=5)
+    r.raise_for_status()
+    return r.text
+
+
+def get_k8s_tags() -> List[str]:
+    """Retrieve semantically ordered k8s releases, newest to oldest."""
+    response = _url_get(K8S_TAGS_URL)
+    tags_json = json.loads(response)
+    if len(tags_json) == 0:
+        raise Exception("No k8s tags retrieved.")
```

Comment:
nit: let's make this exception a bit more specific. Maybe a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError)?